pkg/routeagent_driver/handlers/calico/calico_suite_test.go

```diff
@@ -135,13 +161,40 @@ func (t *testDriver) getDefaultIPPoolIPIPMode() string {
 	return string(p.Spec.IPIPMode)
 }
 
+func (t *testDriver) getDefaultInstallationEncapsulation() string {
+
+	installation, err := calico.GetDefaultInstallation(t.dynClient, gvrInstallations)
```

Comment:
As with `patchCalicoNetworkIpPoolsEncapsulation` below, let's avoid using the production code here:

```suggestion
	installation := test.GetResource(t.dynClient.Resource(gvrInstallations), &tigerav1.Installation{ObjectMeta: metav1.ObjectMeta{
		Name: "default",
	}})
```

Reply:
Changed.Thx